ui/admin/tests/helpers/features-service.js

```diff
@@ -0,0 +1,13 @@
+import { getContext } from '@ember/test-helpers';
+
+export function enableFeature(flag) {
```

Comment:
Do we need to re-create this function? It technically exists currently in the [addon](https://github.com/kategengler/ember-feature-flags/blob/v6.0.0/addon-test-support/index.js)

Reply:
@ZedLi I assume, @cameronperera correct me if I am wrong, adding this test helper will deprecate the `ember-feature-flags`? So we can stop using such package?

Reply:
No, we'd still need the addon to handle the feature flag service functionality, this is just for a test helper that the addon hasn't released yet

Reply:
@ZedLi is correct, this does not remove the need for the addon. This just helps with toggling flags in our tests.

To your original question, my understanding is we had moved to just enabling and disabling feature flags ourselves instead of having a mix of the `enableFeature('flag')` from the addon and `featuresService.disable('flag')`. I don't think we are even using the `enableFeature` helper from the addon. From what I can tell we only use our current method. Happy to remove this `enableFeature` test-helper if the team prefers that route.

Reply:
Gotcha! Understood, thanks for clarification folks 😉

Reply:
As per @ZedLi comment and seeing Cameron answer I think Cameron is right, we aren't using `enableFeature` from the addon. I do not have a strong opinion but I would leave it becasue seems consistent to me our service has `enableFleature` and `disableFeature`. But as I said, no strong opinion 😉

Reply:
It's not a big deal since it's such a simple function, but I lean towards using the existing functionality since it's the exact same function and should have been provided by the addon for this service.

I think we didn't use the `enableFeature` initially because we didn't re-create the helper for the `disableFeature` so it wasn't symmetrical, but I think it makes sense for us to include the helper for `disableFeature` and maybe add a comment why we made this helper and not the `enableFeature` for context.